magefiles/magefile.go

```diff
@@ -226,3 +237,53 @@ func (CI) isPRPairingRequired() bool {
 
 	return false
 }
+
+func (CI) sendWebhook() error {
+	// TODO we will need to distinguish between webhookTargets for AppStudio QE and HACBS QE
+	const saltSecret = "123456789"
```

Comment:
You're removing the config and hardcoding the current AppStudio QE stuff. I'm not necessarily a fan of hard coding this type of data. It would make sense to make it configurable through a config or env vars. What is your plan to be able to override the hardcoded stuff for HACBS QE specific target and jobs?

Reply:
Since all hacbs/appstudio e2e jobs will use centralized scripts, there are only 2 places where we could store this type of data. Here, or in [reusable openshift CI step](https://github.com/openshift/release/blob/master/ci-operator/step-registry/redhat-appstudio/e2e/redhat-appstudio-e2e-commands.sh) (about to be used by all e2e jobs). Would you see that CI step to be more convenient for storing the data about the webhook config?

Reply:
Ah, ok thank you for the information! I'm leaning towards that reusable step would be the proper area. That way even if I try to run this outside of a step I could expose the env variables. Maybe a mixture of both, first look for the env variables and then if none is found or has an empty value default to the AppStudio QE values.

Reply:
updated in https://github.com/redhat-appstudio/e2e-tests/pull/204/commits/0893adbc98b0e1ef5ae5bbd07b34bb0957680c0f